### PR TITLE
sample update

### PR DESCRIPTION
**Pull request by wangjun-microsoft**
show facial hair and head pose detection result in the sample

### CODE REVIEW

File: Sample-WPF/Controls/FaceDetectionPage.xaml [hunk 1]
```diff
@@ -127,8 +127,10 @@
                             <TextBlock Grid.Column="2" Grid.Row="0" HorizontalAlignment="Left" VerticalAlignment="Top" Text="{Binding Gender}" />
                             <TextBlock Grid.Column="2" Grid.Row="0" HorizontalAlignment="Right" VerticalAlignment="Top" Text="{Binding Age}" />
                             <TextBlock Grid.Column="2" Grid.Row="1" HorizontalAlignment="Left" VerticalAlignment="Top" Text="{Binding IsSmiling}" />
-                            <TextBlock Grid.Column="2" Grid.Row="2" Grid.RowSpan="2" HorizontalAlignment="Left" VerticalAlignment="Top" Text="{Binding Glasses}" TextWrapping="Wrap" />
-                        </Grid>
+                            <TextBlock Grid.Column="2" Grid.Row="1" HorizontalAlignment="Right" VerticalAlignment="Top" Text="{Binding Glasses}" TextWrapping="Wrap" />
+                        <TextBlock Grid.Column="2" Grid.Row="2" VerticalAlignment="Top" Text="{Binding FacialHair}" TextWrapping="Wrap" />
```
Comment (huxuan) [inline Sample-WPF/Controls/FaceDetectionPage.xaml]: Indentation, ditto for the following three lines.

Comment (wangjun-microsoft) [inline Sample-WPF/Controls/FaceDetectionPage.xaml]: sure

File: Sample-WPF/Controls/Face.cs [hunk 2]
```diff
@@ -98,6 +98,10 @@ public class Face : INotifyPropertyChanged
         /// </summary>
         private string _glasses;
 
+        /// <summary>
+        /// Indicates the headPose
+        /// </summary>
+        private string _headPose;
```
Comment (wangjun-microsoft) [inline Sample-WPF/Controls/Face.cs]: where to add? after Line 104? or Line 103?

**Comment by huxuan** [comment]
Looks good to me except for several minor style issues.

File: Sample-WPF/Controls/Face.cs (hunk 2 above)
Comment (huxuan) [inline Sample-WPF/Controls/Face.cs]: Before `#endregion Fields`.

**Pull request merged**